test/styles.ts

```diff
@@ -0,0 +1,35 @@
+import styled from '@emotion/styled';
+import { PropsWithFintechTheme } from '@xsite-ui/fintech.theme';
+
+export const FeesWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 266px;
+  flex-shrink: 0;
+`;
+
+export const Title = styled.div<PropsWithFintechTheme>`
+  font-weight: ${({ theme }) => theme?.fontWeights?.semiBold};
+  line-height: 22px;
+  margin-bottom: 8px;
+`;
+
+export const ReqularText = styled.div<PropsWithFintechTheme>`
+  font-size: 14px;
+  line-height: 24px;
+`;
+
+export const RowWrapper = styled.div`
+  justify-content: space-between;
```

Comment:
@openai will it work without `display: flex`?

Reply:
:robot: OpenAI

alex-dolbik: @openai The `display: flex` property is used to create a flex container. Without it, the flex properties such as `flex-direction` and `justify-content` will not work. So, in this case, removing `display: flex` from `RowWrapper` may cause unexpected behavior.

<!-- This is an auto-generated reply by OpenAI -->

Reply:
@openai But we don't have it here? Leave comment, that it should be added

Reply:
:robot: OpenAI

@alex-dolbik, you are correct. The `display: flex` property is not used in the `RowWrapper` component. However, it might be a good idea to add a comment indicating that it should be added if needed in the future.

<!-- This is an auto-generated reply by OpenAI -->